src/controllers/auth/forgetPassword/forgetPassword.js

```diff
@@ -0,0 +1,32 @@
+import Users from '../../../db/models/users';
+import Verifications from '../../../db/models/verifications';
+import mail from '../../../lib/mail';
+
+const forgetPassword = async (req, res) => {
+  const { email } = req.body;
+
+  if (!email) {
+    res.status(400).end();
+
+    return;
+  }
+
+  const user = await Users.findOne({ where: { email } });
+
+  if (!user) {
+    res.status(201).end();
+
+    return;
+  }
+
+  const token = await Verifications.createScopedTokenForUser(
+    user.id,
+    Verifications.SCOPE.RESET_PASSWORD,
+  );
+
+  await mail.sendEmailVerification(email, token);
```

Comment:
`sendEmailVerification` 是发送验证邮箱邮件

```suggestion
  await mail.sendResetPassword(email, token);
```

Reply:
好了!